CanvasPlusPlayground/Common/Network/CanvasRequest.swift

```diff
@@ -28,7 +28,12 @@ enum CanvasRequest: Hashable {
     case getEnrollments(courseId: String, perPage: String = "100")
     
     case getQuizzes(courseId: String, searchTerm: String? = nil)
-    
+
+    /// Pass in `nil` as `userID` to fetch the current user.
+    case getUser(userID: String? = nil)
```

Comment:
better as a default value than Optional (`userID: String = "self"`)

update: i see why u did optionals, feel free to disregard